.eslintrc.js

```diff
@@ -1,17 +1,61 @@
 const { overrides } = require('@netlify/eslint-config-node')
 
 module.exports = {
-  extends: '@netlify/eslint-config-node',
+  extends: ['@netlify/eslint-config-node', 'plugin:node/recommended'],
   rules: {
+    'node/handle-callback-err': 2,
+    'node/no-new-require': 2,
+    'node/exports-style': 2,
+    'node/file-extension-in-import': 2,
+    'node/no-mixed-requires': 2,
+    // Browser globals should not use `require()`. Non-browser globals should
+    'node/prefer-global/console': 2,
+    'node/prefer-global/text-decoder': 2,
+    'node/prefer-global/text-encoder': 2,
+    // This does not work well in a monorepo
+    'node/shebang': 0,
+
+    // TODO: enable this
+    'no-process-exit': 0,
+    // 'node/no-sync': 2,
+    // 'node/callback-return': 2,
+    // 'node/global-require': 2,
+    // 'node/prefer-global/url-search-params': 2,
+    // 'node/prefer-global/url': 2,
+    // 'node/prefer-global/buffer': [2, 'never'],
+    // 'node/prefer-global/process': [2, 'never'],
+
     // TODO: harmonize with filename snake_case in other Netlify Dev projects
     'unicorn/filename-case': [2, { case: 'kebabCase', ignore: ['.*.md'] }],
   },
   overrides: [
     ...overrides,
+    // Documentation site's browser JavaScript
+    {
+      files: ['site/src/**/*.js'],
+      parserOptions: {
+        sourceType: 'module',
+      },
+      rules: {
+        'node/no-unsupported-features/es-syntax': 0,
+        // TODO: enable
+        'node/no-extraneous-import': 0,
```

Comment:
`site/src/**/*.js` use many `import` to dependencies that seem to be indirect dependencies, but not direct dependencies: `rebass`, `react`, `styled-system`, `react-router-dom`, `prop-types`, `lodash.sortby`, `@rebass/markdown`.
Should we add those to `package.json`?

Reply:
Fixed in https://github.com/netlify/cli/pull/1472/commits/2e8ebaa77814bb1fea31d26a884c21951663e226

Reply:
Thanks! :tada: